modules/check_in/app/serializers/check_in/vaos/appointment_serializer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module CheckIn
+  module VAOS
+    class AppointmentSerializer
+      include JSONAPI::Serializer
+
+      set_id(&:id)
+      set_type :appointments
+
+      attributes :kind, :status, :serviceType, :locationId, :clinic, :start, :end, :minutesDuration
```

Comment:
Should cancellable be added to the attributes?

Reply:
for our requirements, we don't have a need for it in check-in experience since these appointments are view-only. any particular reason you think it should be included?

Reply:
I was just making sure! Thanks!